packages/vscode-xml-tools-client/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
tslint is deprecated.
consider if you want to spend time on this or just use JavaScript instead.

Reply:
note your client has tslint but your server does not